src/gui/ScreenHandler.cpp

```diff
@@ -198,9 +240,16 @@ void Screens::Loop() {
                 }
             }
         }
+
+        /// when msgbox(IDialog) is closed on Event and in response is closed
```

Comment:
msgbox is not screen, it is dialog - it is not stored here
it calls GuiLoop so events work, but it never gets into ScreenHandler
Also it auto restores capture when destroyed, so your comment is wrong

Reply:
fixed